config/locales/provider_interface/reference_status_report_service.yml

```diff
@@ -0,0 +1,14 @@
+en:
+  provider_interface:
+    reference_status_report_service:
+      headers:
+        - 'Name'
+        - 'Status'
+        - 'Referee Name'
+        - 'Application Choice ID'
+        - 'Link'
```

Comment:
I'll defer to @MylesJarvis on this one but I think we need to revisit the layout of the report. I don't think we should repeat the candidates name, id and link. Potentially we could also combine these three so it's just the candidates name and that acts as a link to their application.

Reply:
I agree with this, now I've had change to review the design and the PR in more detail. I'll add a full review comment today to elaborate.